src/Mocks/VisualStudio/project.15.0.json

```diff
@@ -5,7 +5,7 @@
     "Microsoft.VisualStudio.Editor": "15.0.25824-RC",
     "Microsoft.VisualStudio.ImageCatalog": "15.0.25824-RC",
     "Microsoft.VisualStudio.ProjectSystem": "15.0.594-pre",
-    "Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime": "14.3.25407",
```

Comment:
That's 15.0 and what do you know, `15.0.25726-Preview5` is the latest. go figure.